README.md

```diff
@@ -57,6 +57,8 @@ it needs to be disabled.
 Download the archive matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it. Then copy the contained binary to your preferred
 location and optionally ensure it is added to your _PATH_.
 
+**Note**: For Windows OS, you can run the binary from any location in the drive where your `%USERPROFILE%` directory is located. Mostly this is in the `C:` drive.
```

Comment:
What does that even mean? Is this even true? Why? If this is a true issue, then I would like to split this out in a separate issue and track this properly. So you are implying that I cannot execute _minishift_ from say the _D_ drive?